simtools/simtel/simtel_config_writer.py

```diff
@@ -46,6 +48,7 @@ def __init__(
         self._label = label
         self._layout_name = layout_name
         self._telescope_model_name = telescope_model_name
+        self.array_trigger_file = None
```

Comment:
Can you explain why this is necessary to define here instead of always writing a file on the fly?

Reply:
We need to know that a array_trigger_file is written. If yes, then we use it for the telescope_model. If note, the array-trigger parameter is not set (e.g., as for optics parameters).